src/classes/book/book.rb

```diff
@@ -0,0 +1,16 @@
+require_relative '../item/item'
+require 'date'
+
+class Book < Item
+  attr_accessor :publisher, :cover_state
+
+  def initialize(publisher, cover_state)
```

Comment:
Kindly add parameters inherited from the parent class to initialize.
![image](https://user-images.githubusercontent.com/106117541/211910231-ecb3959c-75a3-47ad-9f0c-e847d9693759.png)

Reply:
Ups!, thanks!

Reply:
@Camilovelag Done!!